tools/git-hooks/pre-commit

```diff
@@ -43,7 +48,7 @@ fi
 cpp_files=$(git diff --cached --name-only --diff-filter=ACMRT | grep -E "\.(c|cpp|inl|h|hpp)$")
 if [[ -n ${cpp_files} ]]; then
 
-    CLANG_CMD="clang-format-10"
+    CLANG_CMD="clang-format-12"
     if ! command -v $CLANG_CMD &> /dev/null
     then
         CLANG_MAJOR_VERSION=$(clang-format --version | sed -n "s/clang-format version \([0-9]*\)\.[0-9]*\.[0-9]*/\1/p")
```

Comment:
Here you should use `CLANG_CMD` instead of `clang-format`.

Could you maybe rename `CLANG_CMD` into `CLANG_FORMAT_CMD` and `CLANG_MAJOR_VERSION` into `CLANG_FORMAT_MAJOR_VERSION`.